packages/redux/src/reducers/serverConfigValues.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ * The plugins that have been loaded
```

Comment:
Update comment - something about the server config values instead of this, which doesn't apply.